src/RileyMaze/userFunction.js

```diff
@@ -0,0 +1,244 @@
+/* exported RMZUserFunction */
+
+const {GObject, Gtk} = imports.gi;
+
+const {Codeview} = imports.codeview;
+
+const FORWARD = 1;
+const UP = 2;
+const DOWN = 3;
+const JUMP = 4;
+const RILEY = {
```

Comment:
According to the API document this should be a JS Proxy so that we can catch invalid property accesses and flag them as unknown instructions. But I think this is actually fine for now, as long as we do the JS Proxy on the game side so that we know when to add a glitched instruction to the game UI.